app/code/community/Pulchritudinous/Queue/data/pulchqueue_setup/data-upgrade-1.0.2-1.0.3.php

```diff
@@ -0,0 +1,25 @@
+<?php
+$adapter    = $this->getConnection();
+$tableName  = $this->getTable('pulchqueue/labour');
+
+$select = $adapter->select()->from(
+    $tableName,
+    ['id', 'payload']
+);
+
+$offset = 0;
+
+while ($result = $adapter->fetchRow($select->limit(1, $offset))) {
+    $id         = $result['id'];
+    $payload    = @unserialize($result['payload']);
+    $newPayload = json_encode((array) $payload);
```

Comment:
What happens if you have php objects in the payload? Like Varien_Object or such.